action.yml

```diff
@@ -52,7 +52,7 @@ inputs:
   promote:
     description: Promote the deployed version to receive all traffic.
     required: false
-    default: true
+    default: "true"
```

Comment:
Do we need quotes? Other action.ymls don't have them
https://github.com/google-github-actions/setup-gcloud/blob/daadedc81d5f9d3c06d2c92f49202a3cc2b919ba/setup-gcloud/action.yml#L59

Reply:
This is technically a linting error. The action passes it in as a string which needs to be parsed anyways.

Reply:
got it